src/serializers/decoders/prev/v11/decode_graph_schema.c

```diff
@@ -53,8 +62,8 @@ static void _RdbLoadFullTextIndex
 
 	if(!already_loaded) {
 		ASSERT(idx != NULL);
-		Index_SetLanguage(idx, language);
-		Index_SetStopwords(idx, stopwords);
+		if(language  != NULL) Index_SetLanguage(idx, language);
+		if(stopwords != NULL) Index_SetStopwords(idx, &stopwords);
```

Comment:
check if this if is needed

Reply:
yes, this is required, if there are no `stopwords` then stopwords will be `NULL` and we don't want to pass NULL to `Index_SetStopwords`

Reply:
Although I believe we can remove the condition for `language`